AutoCollection/CorrelationContextManager.ts

```diff
@@ -1,6 +1,4 @@
-import http = require("http");
```

Comment:
Not used?

Reply:
Yes it seems like it wasn't being used so I cleaned it up.